### PR TITLE
Feature/tx query

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

related to this - but it doesn't close this issue:
[Add a link to the GitHub issue.](https://github.com/HausDAO/daohaus-monorepo/issues/57)

## Changes

- new endpoint to query on tx

## Packages Added

n/a

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/dao-data/src/Query.ts [hunk 1]
```diff
@@ -317,7 +320,7 @@ export default class Query {
     }
   }
 
-  public async findLatestTransaction({
+  public async findLatestTransactionByDao({
```
Comment (alexkeating) [inline libs/dao-data/src/Query.ts:323]: Should we allow clients to pass in the where clause?

Comment (skuhlmann) [reply libs/dao-data/src/Query.ts:323]: Yeah that's fair. this was a super scoped function for post proposal tx query... but was anticipating a usecase we might not need.

going to convert to a more typical list query and we can pass the where, sort order and limit like this is doing.

Comment (skuhlmann) [reply libs/dao-data/src/Query.ts]: @alexkeating  new commit has this. much cleaner and more flexible. thanks!

File: libs/dao-data/src/Query.ts [hunk 2]
```diff
@@ -317,13 +358,13 @@ export default class Query {
     }
   }
 
-  public async findLatestTransaction({
+  public async findTransaction({
```
Comment (jordanlesich) [inline libs/dao-data/src/Query.ts:361]: This looks like it works for me.

**Review by jordanlesich — APPROVED** [review]
Not a whole lot to report here that already hasn't been said/fixed already. I'd be happy merging this in

**Pull request merged**